src/obs-progress.cpp

```diff
@@ -71,14 +72,12 @@ void startTimer()
 
 void stopTimer()
 {
-	try
+	if (_timer->isActive())
 	{
-		//_timer->stop();
-	}
-	catch (...)
-	{
-		// Empty
+		_timer->stop();
 	}
+	_sources.clear();
+	obs_source_release(_currentSceneSource);
```

Comment:
It looks like there's an extra `release` happening somewhere. Maybe the plugin gets unloaded multiple times for some reason? That might also explain the timer crash that you fixed.

```suggestion
	if (_currentSceneSource)
	{
		obs_source_release(_currentSceneSource);
		_currentSceneSource = NULL;
	}
```